Classes/Resolver.php

```diff
@@ -4,7 +4,7 @@
 use GraphQL\Error\Error;
 use GraphQL\Type\Definition\ResolveInfo;
 
-class Resolver
+abstract class Resolver implements ResolverInterface
```

Comment:
Abstract classes should be prefixed with `Abstract` according to the Flow [CGL](http://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartV/CodingGuideLines/PHP.html) (that I follow mostly)

Reply:
On the one hand this class should be `abstract`, because it cannot be used without subclassing it, on the other hand a name change would be a breaking change. Your call ;)

Reply:
No release = no breaking change ;)